src/app/templates/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM ubuntu:14.04.3
+
+MAINTAINER Kyle Campbell <kc@kc.io>
+
+ENV TERM xterm
+
+RUN useradd -m docker && echo "docker:docker" | chpasswd && adduser docker sudo
+
+RUN apt-get update
+RUN apt-get -y install sudo
+
+RUN apt-get install -y build-essential \
```

Comment:
Didn't want to go with the base node-alpine or something? https://github.com/nodejs/docker-node/blob/90d5e3df903b830d039d3fe8f30e3a62395db37e/7.5/alpine/Dockerfile

Do you know what the image size is looking like?